docs/saving_graphs_for_neural_lam.ipynb

```diff
@@ -0,0 +1,372 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Saving graphs for neural-lam\n",
+    "\n",
+    "From the first tagged version of [neural-lam](https://github.com/mllam/neural-lam) ([v0.1.0](https://github.com/mllam/neural-lam/tree/v0.1.0) and currently, as of `v0.4.0`) has a very specific format that the saved graphs are expected to be in based on pickled pytorch tensors. This notebook describes how to save graphs in that format and what the format is."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weather_model_graphs as wmg\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import torch\n",
+    "import networkx as nx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will start by creating some a graph from some randomly generated latitudes and longitudes:"
```

Comment:
When we feed coordinates like this, without a specified CRS, these are not necessarily assumed to be lat-lons.
```suggestion
    "We will start by creating some a graph from some randomly generated coordinates:"
```